docs/source/architecture/character.rst

```diff
@@ -40,9 +40,17 @@ When Alice wants to share this information with Bob, she can create a
 policy in the NuCypher Network to grant access to him. To grant access to Bob, Alice must:
 
 #. Create a re-encryption key based on the asymmetric private key and Bob's public key
-#. Configure the conditions of the policy e.g. *expiration time, m-of-n threshold values*
-#. Deploy the policy including associated fees to the blockchain
-#. Deploy `n` policy arrangements, each containing a re-encryption key fragment, `kFrag`, to `n` nodes on the NuCypher Network
+#. Determine `n` Ursulas from the NuCypher Network to use in the policy
+#. Configure the conditions of the policy, e.g. *expiration time, m-of-n threshold values*
+#. Deploy the policy to the blockchain and escrow associated policy fees
+#. Create a `treasure map`, encrypted for Bob that contains the list of `n` Ursulas in the policy,
+   and the `n` re-encryption key fragments (`kFrags`), each encrypted for a different Ursula in
+   the policy.
+
+   .. note::
+
+      Even though Bob can decrypt the encrypted treasure map, since each re-encryption fragment is encrypted
+      for a specific Ursula, Bob never has access to the plaintext re-encryption fragment.
```

Comment:
```suggestion
      Bob can decrypt the encrypted treasure map, but since each re-encryption fragment is encrypted
      for a specific Ursula, Bob never has access to any re-encryption fragment in plaintext form.
```
Otherwise it reads a bit weird – i.e. that Bob can decrypt the treasure map because the fragments are encrypted on a per-Ursula basis